src/plugins/intel_gpu/src/kernel_selector/cl_kernels/strided_slice_ref.cl

```diff
@@ -326,7 +326,7 @@ KERNEL(strided_slice_ref)(OPTIONAL_SHAPE_INFO_ARG
     FUSED_OPS;
     output[output_index] = FUSED_OPS_RESULT;
 #else
-    output[output_index] = ACTIVATION(input[input_index], ACTIVATION_PARAMS);
+    output[output_index] = ACTIVATION((UNIT_TYPE)input[input_index], (UNIT_TYPE)ACTIVATION_PARAMS);
```

Comment:
Don't use unit type. Cast to either OUTPUT_TYPE or some kind of ACCUMULATOR_TYPE

Reply:
I'll fix it. thanks.

Reply:
Fixed it. please review it again.